src/bridge/fetch.ts

```diff
@@ -136,6 +140,11 @@ registerBridge('fetch', function fetchBridge(ctx: Context, bridge: Bridge, urlSt
     try {
       res.pause()
 
+      let zipStream
+      if (inflate && res.headers["content-encoding"] == "gzip") {
+        zipStream = zlib.createUnzip()
```

Comment:
You want `zlib.createGunzip()` here. zip and gzip are very different :)

Reply:
`zlib.createUnzip` is correct here, it detects deflate/gzip and does the right thing. It's just an awkwardly named API.